src/seq_mv/csr_matrix.c

```diff
@@ -333,18 +331,21 @@ hypre_CSRMatrixSetDataOwner( hypre_CSRMatrix *matrix,
 HYPRE_Int
 hypre_CSRMatrixSetRownnzHost( hypre_CSRMatrix *matrix )
 {
-   HYPRE_Int  ierr = 0;
-   HYPRE_Int  num_rows = hypre_CSRMatrixNumRows(matrix);
-   HYPRE_Int *A_i = hypre_CSRMatrixI(matrix);
-   HYPRE_Int *Arownnz;
+   HYPRE_Int   num_rows = hypre_CSRMatrixNumRows(matrix);
+   HYPRE_Int  *A_i = hypre_CSRMatrixI(matrix);
+   HYPRE_Int  *Arownnz = hypre_CSRMatrixRownnz(matrix);
 
-   HYPRE_Int i, adiag;
-   HYPRE_Int irownnz = 0;
+   HYPRE_Int   i;
+   HYPRE_Int   irownnz = 0;
+
+   if (Arownnz)
```

Comment:
remove `if (Arownnz)`

Reply:
Thanks for catching it! Fixed with [f73e500](https://github.com/hypre-space/hypre/pull/599/commits/f73e50096c721bebdce8372b71174a391c56c98f)